.github/workflows/ci.yml

```diff
@@ -58,6 +58,21 @@ jobs:
       - name: test
         run: make test TEST_ARGS="-vv --showlocals"
 
+      - name: test (timestamp-authority)
+        if: ${{ matrix.conf.os == 'ubuntu-latest' }}
+        run: |
+          wget https://github.com/sigstore/timestamp-authority/releases/download/${SIGSTORE_TIMESTAMP}/timestamp-server-linux-amd64 -O /tmp/timestamp-server
+          chmod +x /tmp/timestamp-server
+          # Run the TSA in background
+          /tmp/timestamp-server serve --port 3000 --disable-ntp-monitoring &
+          export TEST_SIGSTORE_TIMESTAMP_AUTHORITY_URL="http://localhost:3000/api/v1/timestamp"
+          # Ensure Timestamp Authority tests are not skipped by
+          # having pytest show skipped tests and verifying ours are running
+          make test TEST_ARGS="-m timestamp_authority -rs" | tee output
+          ! grep -q "skipping test that requires a Timestamp Authority" output || (echo "ERROR: Found skip message" && exit 1)
+        env:
+          SIGSTORE_TIMESTAMP: "v1.2.3"
```

Comment:
I don't _love_ that we pull down a binary here, and hardcode the version. It'd be great if we could get this into Dependabot somehow.

(Not a blocker, flagging as a follow-up.)